src/Libraries/PythonNodeModels/PythonNode.cs

```diff
@@ -56,13 +72,30 @@ protected override string GetInputTooltip(int index)
             var vals = additionalBindings.Select(x => x.Item2).ToList();
             vals.Add(AstFactory.BuildExprList(inputAstNodes));
 
-            Func<string, IList, IList, object> backendMethod =
-                IronPythonEvaluator.EvaluateIronPythonScript;
+            
+            if (String.IsNullOrEmpty(Engine))
+            { 
+                Engine = DefaultPythonEngine;
+            }
+
+            Func<string, IList, IList, object> pythonEvaluatorMethod;
+            if (Engine == "CPython3")
```

Comment:
Can we add a const string for "CPython3" as well.

Reply:
Please do, I have dependency on this..

Reply:
Added it.